audb/core/load.py

```diff
@@ -741,6 +765,10 @@ def load(
     Returns:
         database object
 
+    Raises:
+        ValueError: if no table in the database matches the requested ones
```

Comment:
Maybe combine to one? E.g.:

```suggestion
        ValueError: if table or media is requested that does not exist
```

Reply:
Changed accordingly and updated description.